clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -564,6 +564,51 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+template <int GROUP_THREADS, typename InputT, int ITEMS_PER_THREAD,
+          typename InputIteratorT>
+__dpct_inline__ void load_blocked(int linear_tid, InputIteratorT block_itr,
```

Comment:
I would like to use `size_t` as the type of linear_id

Reply:
Added